mocha/utils/apiEndpoints.js

```diff
@@ -3,3 +3,4 @@ exports.feedbackEndpoint = "/api/feedback";
 exports.signUpApiEndpoint = "/api/auth/signup";
 exports.organisationsEndpoint = "/api/organisations";
 exports.postsEndpoint = "/api/posts";
+exports.versionApiEndpoint = "/api/version";
```

Comment:
@adorapham @jessica-lau 
what naming convention do you prefer? With or without APi? signUp**Api**Endpoint or signUpEndpoint?
Maybe we could unify that?

Reply:
I prefer ApiEndpoint seem it is a short of for the path  for example signUpApiEndpoint  /api/auth/signup.

Reply:
I personally prefer without "Api" in the names as it keeps them simple and short.

Reply:
Ok, so I will update all Endpoint without API and associated test.js files to this PR

Reply:
@jessica-lau @galeza , I am done with the updated , please check